src/app/utils/api/fetchMovies.ts

```diff
@@ -0,0 +1,7 @@
+import { Movie } from "@prisma/client";
+import { PaginationSchema } from "../paginationFilter";
+
+export const fetchMovies = async ({ limit, page }: PaginationSchema) => {
+  const result = await fetch(`/api/movies/all?limit=${limit}&page=${page}`);
```

Comment:
The fetch call's promise resolves with a `response` ✅ not `result` ❌

Reply:
Updated! :)